packages/components/psammead-brand/src/index.test.jsx

```diff
@@ -1,10 +1,27 @@
 import React from 'react';
 import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
+import * as svgs from '@bbc/psammead-assets/svgs';
 import Brand from './index';
 
 describe('Brand', () => {
   shouldMatchSnapshot(
-    'should render correctly',
+    'should render correctly with default News service svg',
     <Brand brandName="Default Brand Name" />,
   );
+  shouldMatchSnapshot(
```

Comment:
I dont think we need to test this using real `@bbc/psammead-assets/svgs` assets. Maybe 1 :P

Reply:
We can also use this in the storybook to avoid pulling in `@bbc/psammead-assets/svgs` completely
```jsx
const svg = {
  group: (
    <g fillrule="evenodd">
      <path d="M84.32" />
    </g>
  ),
  viewbox: {
    height: 24,
    width: 167.95
  }
};
```
What is in the svg doesn't matter provided it is rendered correctly within `Brand`